ooni/nettest.py

```diff
@@ -591,11 +594,16 @@ def generateMeasurements(self):
                 test_instance._setUp()
                 test_instance.summary = self.summary
                 for method in test_methods:
+                    try:
+                        measurement = self.makeMeasurement(
+                            test_instance,
+                            method,
+                            test_input)
+                    except Exception:
+                        log.exception(failure.Failure())
```

Comment:
I'd rather be explicit about failure creation. `log.exception(None)` does not look like _write current exception information_.

Moreover, it make some sense to occasionally review all `log.exception` calls to check if lost traceback is ok for this case.